crates/hotshot/examples/web-server-da/multi-validator.rs

```diff
@@ -3,7 +3,7 @@ use async_compatibility_layer::{
     logging::{setup_backtrace, setup_logging},
 };
 use clap::Parser;
-use hotshot::demos::sdemo::SDemoTypes;
+use hotshot::demo::SDemoTypes;
```

Comment:
Since we renamed `sdemo` to `demo` maybe we should also change `SDemoTypes` to `DemoTypes`?